scripts/DBSS-JP/c100414018.lua

```diff
@@ -47,6 +47,7 @@ function c100414018.atkop(e,tp,eg,ep,ev,re,r,rp)
 		e1:SetType(EFFECT_TYPE_SINGLE)
 		e1:SetCode(EFFECT_UPDATE_ATTACK)
 		e1:SetValue(1000)
+		e1:SetProperty(EFFECT_FLAG_CANNOT_DISABLE)

```

Comment:
no need to add this flag to monster effects which never apply to itself

Reply:
are there other cards need to revert?